build/buildSrc/build.gradle

```diff
@@ -19,27 +19,40 @@
 // along with hale-build.  If not, see <http://www.gnu.org/licenses/>.
 
 repositories {
+	mavenLocal()
```

Comment:
For what is mavenLocal used here? Seems like this could different behavior on different systems, especially when used as first entry. Can we remove it again?

Reply:
Not sure why it was added but it can safely be removed. I have removed it.